### PR TITLE
Corrected out-of-service requests for xml files

### PR DESCRIPTION
**Pull request by tomasg2012**
Tool incorrectly appends target host to external schema definitions, causing errors in resolution.

Signed-off-by: Tomas <tomas.g.2012@gmail.com>

### CODE REVIEW

**Comment by jautor** [comment]
Ensure that the release notes include comments about the new requirement for Requests library (shouldn't be a problem as we use it in a lot of tools, and a popular package)

**Comment by tomasg2012** [comment]
Seems the readme and requirements already document this.

File: common/traverse.py [hunk 1]
```diff
@@ -158,7 +159,11 @@ def callResourceURI(self, URILink):
         try:
             startTick = datetime.now()
             mockup_file_path = os.path.join(config['mockup'], URLDest.replace('/redfish/v1/', '', 1).strip('/'), 'index.json')
-            if config['mockup'] != '' and os.path.isfile(mockup_file_path):
+            if not inService:
+                req = requests.get(URLDest)
```
Comment (mraineri) [inline common/traverse.py:163]: Need to add proxy parameter to this call like how it's done with the schema bundle download call.

**Comment by tomasg2012** [comment]
This runs on my end, but Mike you might wanna confirm that `._proxies` is the same proxies passed into the `redfish_client`

**Comment by mraineri** [comment]
Whoops, missed your update. Completely forgot we tracked the proxies like that, so that made it easy!

**Pull request merged**